example/web/index.html

```diff
@@ -0,0 +1,104 @@
+<!DOCTYPE html>
```

Comment:
I don't think we support Web, can we remove "Web"?

Reply:
Removed it.